schema205/md/schema_table.py

```diff
@@ -57,7 +57,8 @@ def data_elements_dict_from_data_groups(data_groups):
             new_obj = data_groups[dat_gr]["Data Elements"][element]
             new_obj["Name"] = f"`{element}`"
             if 'Required' in new_obj:
-                new_obj["Req"] = u'\N{check mark}' if new_obj["Required"] else ''
+                check = u'\N{check mark}'
```

Comment:
Just a note that I had to work around issues with the Unicode checkmark and the 205 docs. If you've found a way to get the checkmark to render cleanly into the PDF, then you're good. If not, you may want to check out the "enhancements-for-docs" branch and specifically https://github.com/open205/schema-205/blob/enhancements-for-docs/schema205/md/schema_table.py#L48

Reply:
I figured out a way to handle the unicode. Thanks!